apps/webcert/src/components/fmb/FMBPanel.tsx

```diff
@@ -20,11 +19,11 @@ export const Italic = styled.p`
 
 const FMBPanel: React.FC = () => {
   const dispatch = useAppDispatch()
-  const fmbDiagnosisCodes = useSelector(getFMBDiagnosisCodes, isEqual)
+  const fmbDiagnosisCodes = useAppSelector(getFMBDiagnosisCodes, isEqual)
   const [selectedDiagnosisCode, setSelectedDiagnosisCode] = useState<FMBDiagnosisCodeInfo>()
-  const diagnosisValue = useSelector(getDiagnosisListValue, isEqual)
+  const diagnosisValue = useAppSelector(getDiagnosisListValue, isEqual)
   const isIcd10Chosen =
-    !diagnosisValue || diagnosisValue.list.length === 0 || diagnosisValue.list[0].terminology.toLowerCase().includes('icd')
+    !diagnosisValue || diagnosisValue.list.length === 0 || diagnosisValue.list[0]?.terminology.toLowerCase().includes('icd')
```

Comment:
This is the most likely cause of the error that I could find without directly reproducing the issue.